integration_tests/__tests__/console-test.js

```diff
@@ -9,20 +9,21 @@
  */
 'use strict';
 
+const path = require('path');
 const runJest = require('../runJest');
-const skipOnWindows = require('skipOnWindows');
-
-skipOnWindows.suite();
+const {normalizePathsInSnapshot} = require('jest-util');
 
 test('console printing', () => {
-  const result = runJest('console');
+  // On Windows, this test only works when --no-cache is passed in. Need to
+  // determine why.
```

Comment:
I feel like this is uncovering a bug somewhere, I had no idea where to look though :/